### DISCUSSION TITLE
Understanding default_redirection_url in Authelia Configuration

### DISCUSSION THREAD

**Discussion by qymab** [authelia/authelia#7430] [Q&A] [answered]
I'm struggling to understand what I should specify for default_redirection_url. Although I have configured the domain and authelia_url, I remain unclear about default_redirection_url.
```
      domain: 'domain.com'
      authelia_url: 'https://auth.domain.com'
      default_redirection_url: 'What should I specify here?'
```

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

The `default_redirection_url` is the URL where users are redirected when Authelia cannot detect the target URL where the user was heading. This typically happens when a user visits the sign-in portal directly without a specific target URL.

Here are some guidelines for setting the `default_redirection_url`:

1. **General Use Case**: If you have a main landing page or a dashboard that users should be redirected to after logging in, you can set that URL as the `default_redirection_url`.
   
   Example:
   ```yaml
   default_redirection_url: 'https://www.domain.com/dashboard'
   ```

2. **Single Application**: If Authelia is primarily used to protect a single application, you can set the application's main URL as the `default_redirection_url`.

   Example:
   ```yaml
   default_redirection_url: 'https://app.domain.com'
   ```

3. **Multiple Applications**: If you have multiple applications, you might want to set a general landing page or a portal that lists all available applications.

   Example:
   ```yaml
   default_redirection_url: 'https://portal.domain.com'
   ```

In summary, the `default_redirection_url` should be a URL that makes sense for your users to land on when they log in without a specific target. This could be a dashboard, a main application, or a portal page.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by hendrik1120** [comment, marked as answer]
Basically what the AI said, but you will find a more detailed explanation in the docs.
https://www.authelia.com/configuration/miscellaneous/introduction/#default_redirection_url
